x-pack/plugins/uptime/public/components/certificates/fingerprint_col.tsx

```diff
@@ -27,7 +27,7 @@ interface Props {
 export const FingerprintCol: React.FC<Props> = ({ cert }) => {
   const ShaComponent = ({ text, val }: { text: string; val: string }) => {
     return (
-      <Span data-test-subj={val}>
+      <Span data-test-subj={val} className="eui-textNoWrap">
```

Comment:
Can we name this `StyledSpan` or `FingerprintSpan`, something like that? I found myself wondering if there was some special new React feature added when I saw this.